controllers/src/main/scala/cromwell/pipeline/controller/ProjectConfigurationController.scala

```diff
@@ -0,0 +1,47 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Route
+import cromwell.pipeline.datastorage.utils.auth.AccessTokenContent
+import cromwell.pipeline.service.ProjectConfigurationService
+import akka.http.scaladsl.server.Directives._
```

Comment:
This is nitpicking, but this import should be placed a little bit higher.
It is just a single comment, not a changes request.